bin/packages/build.js

```diff
@@ -19,7 +19,7 @@ const PACKAGES_DIR = path.resolve( __dirname, '../../packages' );
 /**
  * Get the package name for a specified file
  *
- * @param  {string} file File name
+ * @param {string} file File name
  * @return {string}      Package name
```

Comment:
If `@return` descriptions are not destined to be aligned with `@param` descriptions then I think this also requires some changes:

This aligns the descriptions of both lines:
```suggestion
 * @param {string} file File name
 * @return {string}     Package name
```

Or, this removes the additional whitespece preceding the @return description

```diff
 * @param {string} file File name
 * @return {string} Package name
```

Possibly, another convention observed further below is a blank line between the last @param and the @raturn:

```diff
 * @param {string} file File name
 *
 * @return {string} Package name
```